test/reduce.js

```diff
@@ -61,9 +61,6 @@ module.exports = function reduce_test(testcase, minify_options, reduce_options)
 
             var parent = tt.parent();
 
-            // ignore call expressions
-            if (parent instanceof U.AST_Call && parent.expression === node) return;
```

Comment:
This was needed once. Apparently not the case now.

Reply:
We can now compare between runtime exceptions properly, so `0()` isn't an issue anymore.